.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,17 @@
+#### Bug or Feature Description
+<!--- * *Describe the problem, ideally from the customer's viewpoint*  -->
 
-#### Affected Platform(s)
-* *Platform (for example CentOS 7.2)*
+#### Description of Your Change
+<!--- * *Say how you fixed the problem.  Please describe your code changes in detail for reviewers* -->
 
-#### Cause / Analysis / Design
-* *If there is an external design, link to it **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)***
-* *If there is a discussion on the Forum, link to it **[pbspro community forum](http://community.pbspro.org/)***
-* *Bugs: briefly describe root cause/analysis of the problem*
+#### Link to Design
+<!--- * *If there is a design, link to it here: **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)*** -->
 
-#### Solution Description
-* *Please describe your code changes in detail for reviewers*
+#### Testing Logs or Output
+<!--- * *Please attach your test log output from running the test you added (or from existing tests that cover your changes)* -->
 
-#### Testing logs/output
-* *Please attach your test log output from running the test you added (or from existing tests that cover your changes)*
-
-#### Checklist:
-<!--- Use the preview button to see the checkboxes/links properly. -->
-<!--- Go over the following points, and put an `x` (without spaces around it) in the boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
-- [ ] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
-- [ ] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
-- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
-   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
-- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
-   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
-- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
-- [ ] I have attached **test logs to this pull request** as evidence of testing/verification.
-
-
-__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__
+#### Pull Request Guidelines
```

Comment:
I think we missed this last night, we should make this whole section a comment since this is just for the developer's own use while they are creating the PR, not something that needs to be visible to their reviewers.